openwisp_monitoring/monitoring/base/models.py

```diff
@@ -475,7 +455,7 @@ def _get_top_fields(self, number, time=DEFAULT_TIME, timezone=settings.TIME_ZONE
         q = self.get_query(
             query=q, summary=True, fields=['SUM(*)'], time=time, timezone=timezone
```

Comment:
what about SUM?

Reply:
I checked few Timeseries dB query languages to realize that many things such as the basic operators, functions (includes sum) are common. Hence, didn't make any changes. Let me know if I should.

Reply:
same logic applies, it's timeseries DB related code which must be delegated to the timeseries DB backend entirely, it should not be left lying around in here.

Let's find a way to do that.